app/scripts/modules/core/delivery/executionGroup/execution/execution.directive.js

```diff
@@ -94,6 +94,15 @@ module.exports = angular
       });
     };
 
+    this.restartDetails = () => {
```

Comment:
not sure if it's wise for this to be a function, or if there's maybe a better way to handle this